Library/Homebrew/test/attestation_spec.rb

```diff
@@ -92,6 +92,46 @@
     end
   end
 
+  # NOTE: `Homebrew::CLI::NamedArgs` will often return frozen arrays of formulae
+  #       so that's why we test with frozen arrays here.
+  describe "::sort_formulae_for_install", :integration_test do
```

Comment:
I know we're always worried about integration tests but this should not be too slow since it's only added here so we can use `setup_test_formula`.

```console
$ brew tests --only=attestation:97 --profile=5
Randomized with seed 54181
/usr/local/Homebrew/Library/Homebrew/api.rb:9: warning: base64 was loaded from the standard library, but will no longer be part of the default gems since Ruby 3.4.0. Add base64 to your Gemfile or gemspec.
Run options:
  include {:locations=>{"./test/attestation_spec.rb"=>[97]}}
  exclude {:needs_linux=>true, :needs_network=>true, :needs_ci=>true, :needs_svn=>true}

Randomized with seed 54181
...

Top 3 slowest examples (0.15487 seconds, 98.0% of total time):
  Homebrew::Attestation::sort_formulae_for_install when `gh` is in the formula list moves `gh` formulae to the front of the list
    0.10629 seconds ./test/attestation_spec.rb:107
  Homebrew::Attestation::sort_formulae_for_install when the formula list is empty checks for the `gh` executable
    0.0278 seconds ./test/attestation_spec.rb:121
  Homebrew::Attestation::sort_formulae_for_install when `gh` is not in the formula list checks for the `gh` executable
    0.02078 seconds ./test/attestation_spec.rb:128

Finished in 0.158 seconds (files took 3.86 seconds to load)
3 examples, 0 failures
```